priv/template/config/dev.exs

```diff
@@ -3,7 +3,8 @@ use Mix.Config
 config :<%= application_name %>, <%= application_module %>.Endpoint,
   http: [port: System.get_env("PORT") || 4000],
   debug_errors: true,
-  cache_static_lookup: false
+  cache_static_lookup: false,
+  reloadable_paths: ["--elixirc-paths", "web"]
```

Comment:
There is no need to add this line to  `dev.exs`. Most people won't configure this, so it could be hidden imo.